onadata/apps/api/tests/viewsets/test_organization_profile_viewset.py

```diff
@@ -1028,3 +1028,58 @@ def test_creator_in_users(self):
             'gravatar': self.user.profile.gravatar
         }
         self.assertIn(expected_user, response.data[0]['users'])
+
+    def test_creator_permissions(self):
+        """
+        Test that the creator of the organization has the necessary
+        permissions
+        """
+        self._org_create()
+        request = self.factory.get('/', **self.extra)
+        response = self.view(request)
+        self.assertNotEqual(response.get('Cache-Control'), None)
+        self.assertEqual(response.status_code, 200)
+
+        orgs = OrganizationProfile.objects.filter(creator=self.user)
+        self.assertEqual(orgs.count(), 1)
+        org = OrganizationProfile.objects.filter(creator=self.user).first()
```

Comment:
Would the following work since `orgs` is already defined in line 1043?
```python
...
org = orgs.first()
```

Reply:
Yes it would, changed this.